core/components/crosscontextssettings/processors/mgr/settings/getvaluelist.class.php

```diff
@@ -58,8 +58,12 @@ public function getData() {
         /* query for chunks */
         $c = $this->modx->newQuery($this->classKey);
         $c = $this->prepareQueryBeforeCount($c);
-//        $data['total'] = $this->modx->getCount($this->classKey, $c);
-        $data['total'] = $this->modx->crosscontextssettings->getQueryCount($this->classKey, $c);
+        $version = $this->modx->getVersionData();
+        if ($version['major_version'] <= 2) {
+            $data['total'] = $this->modx->crosscontextssettings->getQueryCount($this->classKey, $c);
+        } else {
+            $data['total'] = $this->modx->getCount($this->classKey, $c);
```

Comment:
this original counting is avoided because xPDO can't handle counting query with function, in this case "groupby()".